policies/rules/security/entry.go

```diff
@@ -0,0 +1,750 @@
+	ans.DestinationZones = util.StrToMem(o.DestinationZones)
+	ans.SourceAddresses = util.StrToMem(o.SourceAddresses)
+	ans.SourceUsers = util.StrToMem(o.SourceUsers)
+	ans.DestinationAddresses = util.StrToMem(o.DestinationAddresses)
+	ans.Services = util.StrToMem(o.Services)
+	ans.Categories = util.StrToMem(o.Categories)
+	ans.Applications = util.StrToMem(o.Applications)
+	ans.SourceDevices = util.StrToMem(o.SourceDevices)
+	ans.DestinationDevices = util.StrToMem(o.DestinationDevices)
+	ans.Schedule = o.Schedule
+	ans.Tags = util.StrToMem(o.Tags)
+	if o.NegateSource != nil {
+		*ans.NegateSource = util.YesNo(*o.NegateSource)
+	}
+	if o.NegateDestination != nil {
+		*ans.NegateDestination = util.YesNo(*o.NegateDestination)
+	}
+	if o.Disabled != nil {
+		*ans.Disabled = util.YesNo(*o.Disabled)
+	}
+	ans.Description = o.Description
+	ans.GroupTag = o.GroupTag
+	ans.Action = o.Action
+	if o.IcmpUnreachable != nil {
+		*ans.IcmpUnreachable = util.YesNo(*o.IcmpUnreachable)
+	}
+	ans.Type = o.Type
+	if o.DisableServerResponseInspection != nil {
+		ans.Options = &secOptions{Misc: o.Misc["secOptions"]}
+		*ans.Options.DisableServerResponseInspection = util.YesNo(*o.DisableServerResponseInspection)
+	}
+	ans.LogSetting = o.LogSetting
+	if o.LogStart != nil {
+		*ans.LogStart = util.YesNo(*o.LogStart)
+	}
+	if o.LogEnd != nil {
+		*ans.LogEnd = util.YesNo(*o.LogEnd)
+	}
+	if o.ProfileSettings != nil {
+		ans.ProfileSettings = &profileSettings{Misc: o.Misc["profileSettings"]}
+		ans.ProfileSettings.Groups = util.StrToMem(o.ProfileSettings.Groups)
+		if o.ProfileSettings.Profiles != nil {
+			ans.ProfileSettings.Profiles = &profileSettingsProfiles{Misc: o.Misc["profileSettingsProfiles"]}
+			ans.ProfileSettings.Profiles.UrlFilteringProfiles = util.StrToMem(o.ProfileSettings.Profiles.UrlFilteringProfiles)
+			ans.ProfileSettings.Profiles.DataFilteringProfiles = util.StrToMem(o.ProfileSettings.Profiles.DataFilteringProfiles)
+			ans.ProfileSettings.Profiles.FileBlockingProfiles = util.StrToMem(o.ProfileSettings.Profiles.FileBlockingProfiles)
+			ans.ProfileSettings.Profiles.WildfireAnalysisProfiles = util.StrToMem(o.ProfileSettings.Profiles.WildfireAnalysisProfiles)
+			ans.ProfileSettings.Profiles.AntiVirusProfiles = util.StrToMem(o.ProfileSettings.Profiles.AntiVirusProfiles)
+			ans.ProfileSettings.Profiles.AntiSpywareProfiles = util.StrToMem(o.ProfileSettings.Profiles.AntiSpywareProfiles)
+			ans.ProfileSettings.Profiles.VulnerabilityProfiles = util.StrToMem(o.ProfileSettings.Profiles.VulnerabilityProfiles)
+		}
+	}
+	if o.Qos != nil {
+		ans.Qos = &qos{Misc: o.Misc["qos"]}
+		ans.Qos.Marking = &qosMarking{Misc: o.Misc["qosMarking"]}
+		ans.Qos.Marking.IpDscp = o.Qos.IpDscp
+		ans.Qos.Marking.IpPrecedence = o.Qos.IpPrecedence
+		if o.Qos.FollowClientToServerFlow != nil {
+			ans.Qos.Marking.FollowClientToServerFlow = ""
+		}
+	}
+
+	return ans, nil
+}
+
+type Entry1Container struct {
+	Answer []Entry1 `xml:"entry"`
```

Comment:
I don't follow what this var is responsible for. (Just naming stuff)

Reply:
Is this comment talking about line 407 or something else?